README.md

```diff
@@ -103,5 +110,15 @@ Until we maybe decide on Celery for running various (timed or otherwise) tasks a
   - `npm run watch &` If you want to develop the JS/LESS stuff this will autocompile them on change
   - `./manage.py runserver 0.0.0.0:8000`
   - `maildump -p ~/maildump.pid --stop`
+  - Make localizations: `find . -name '._*' | xargs rm ; for app in $( find . -path '*/locale'); do (cd $(dirname $app) && ../manage.py makemessages ); done`
```

Comment:
"for app in $( find . -path '*/locale');" was fixed with "for app in locale */locale;" in another chunk in this pull request. Should this be too?